Sites/PR4/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>GD2 PR4</title>
+    <link rel="stylesheet" href="Styles.css">
+
+</head>
+
+<body>
+    <div class="containerblue clearfix">
+
+
+        <h1><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 146.19 78.17">
+
+                <title>Asset 1</title>
+                <g id="Layer_2" data-name="Layer 2">
+                    <g id="Layer_1-2" data-name="Layer 1"><text class="alternate" transform="translate(0.22 30.85)">
+                            <tspan class="transparent">AB</tspan>
+                            <tspan class="transparent" x="0" y="30">C</tspan>
+                            <tspan x="26.68" y="30">Design</tspan>
+                        </text></g>
+                </g>
+            </svg></h1>
+        <span id="closer">
+            <h2>Actively</h2>
+            <h2>Brainstorming</h2>
+            <h2>Calculated</h2>
+            <h2>Design</h2>
+        </span>
+        &nbsp
+        <h3> A conference centered around an active effort to make good design choices for ergonomics and user
+            preference.
+            Beautiful interfaces mean nothing if we can't answer the "what" and "why" of design. </h3>
+        <div> &nbsp </div> <br>
+        <a href="#" class="bold"> Home </a>
+        &nbsp / &nbsp <a href="learnmore.html">Learn More</a> &nbsp / &nbsp
+        <a href="registernow.html">Register
+            Now </a>
+
+
+</body>
```

Comment:
You never close your container `div`:
```suggestion
    </div>
</body>
```